lib/Experian/IDAuth.pm

```diff
@@ -309,7 +309,13 @@ sub _get_result_proveid {
 
     return unless $credit_reference and $kyc_summary;
 
-    my $decision = {matches => []};
+    my $decision = {matches => [], kyc_summary_score => 0};
+
+    # calculate kyc summary score
+    if ($kyc_summary->findvalue('FullNameAndAddress/Count') > 0 && $kyc_summary->findvalue('DateOfBirth/Count') > 0)
```

Comment:
Will `findvalue()` return negative numbers here?  If it won't, maybe this could be simplified to just

```perl
if ($kyc_summary->findvalue('FullNameAndAddress/Count') && $kyc_summary->findvalue('DateOfBirth/Count'))
{
    ...
}
```

Reply:
Yeap negative values are a possibility unfortunately